site/index.html

```diff
@@ -50,6 +50,7 @@ <h1><img src="img/logo.png" alt="Cleveland Tip Jar" width="300px"></h1>
         Every time you have a drink or a take-out meal at home during social
         distancing, consider tipping a local worker through Venmo, PayPal, or Cash App.
       </p>
+      <p>Service workers include but are not limited to those in the food service, hospitality service, personal care service, and other related industries.</p>
```

Comment:
Maybe we could move this down next to the signup link. E.g.: "Service workers: add yourself to this list. All workers who normally rely on tips are welcome."

Reply:
done. i'll commit it in a second